dali/operators/generic/one_hot.cc

```diff
@@ -67,9 +67,9 @@ void OneHot::RunImpl(HostWorkspace &ws) {
                                                   off_value_);
         });
       }
+      tp.WaitForWork();
```

Comment:
Why it needs to be placed inside the switch?

Reply:
otherwise the views go out of the scope and it might throw a segfault